pkgs/tools/inputmethods/fcitx5/fcitx5-skk.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv
+, fetchFromGitHub
+, pkg-config
+, cmake
+, extra-cmake-modules
+, gettext
+, fcitx5
+, fcitx5-qt
+, libskk
+, qtbase
```

Comment:
> Qt5 (optional), for rule and dictionary configuration UI.
According to the documentation of fcitx5-skk, qtbase is optional with cmake flag `ENABLE_QT`. 

By the way, should be qt6 added as well?

Reply:
Fixed.

It seems we have to add `fcitx5-qt` in https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/qt6-packages.nix before build `fcitx5-skk` with Qt6.